python/ray/rllib/agents/impala/vtrace_policy_graph.py

```diff
@@ -292,6 +308,19 @@ def make_time_major(tensor, drop_last=False):
             max_seq_len=self.config["model"]["max_seq_len"],
             batch_divisibility_req=self.config["sample_batch_size"])
 
+        pi_loss_abs = tf.abs(self.loss.pi_loss)
```

Comment:
if some of these are generally important stats for impala we might consider pushing them upstream, they are very grateful for each our contribution

Reply:
These are general RL debugging metrics, not impala-specific.